core/templates/home.html

```diff
@@ -6,30 +6,32 @@ <h2> teste </h2>
 {% endblock %}
 
 {% block main %}
-<main class="text-center mb-10 mx-auto mt-8">
-  <div class="grid gap-2 grid-cols-1 mb-4">
-    <h1>Bem-vinda</h1>
-    <p class="max-w-xs m-auto">O cadastro leva cerca de 7 minutos, seguido por uma capacitação online em violência de
-      gênero. Vamos lá?</p>
-  </div>
-  <div class="grid gap-3 grid-cols-1 w-64 mx-auto mb-6 text-lg">
-    <a href="{% url 'forms_steps' type_form='psicologa' step=1 %}" class="btn btn-secondary">Sou psicóloga</a>
-    <a href="{% url 'forms_steps' type_form='advogada' step=1 %}" class="btn btn-primary">Sou advogada</a>
-  </div>
-  <p class="text-sm  m-auto max-w-xs">Ao realizar o cadastro você está de acordo com a nossa <a href="#">política de
-      privacidade</a>.</p>
 
-</main>
-
-{% endblock %}
+<div class="grid grid-cols-1 md:grid-cols-2 xs:justify-center w-full gap-5 text-center">
+  <div class="max-w-lg mt-44 mx-auto">
+    <div class="grid gap-2 grid-cols-1 mb-4 ">
+      <h1>Bem-vinda</h1>
+      <p class="max-w-xs m-auto">O cadastro leva cerca de 7 minutos, seguido por uma capacitação online em violência de
+        gênero. Vamos lá?</p>
+    </div>
+    <div class="grid gap-3 grid-cols-1 w-64 mx-auto mb-6 text-lg">
+      <a href="{% url 'forms_steps' type_form='psicologa' step=1 %}" class="btn btn-secondary">Sou psicóloga</a>
+      <a href="{% url 'forms_steps' type_form='advogada' step=1 %}" class="btn btn-primary">Sou advogada</a>
+    </div>
+    <p class="text-sm  m-auto max-w-xs">Ao realizar o cadastro você está de acordo com a nossa <a href="#">política de
```

Comment:
```suggestion
    <p class="text-sm m-auto max-w-xs">Ao realizar o cadastro você está de acordo com a nossa <a href="#">política de
```

esse link da política de privacidade deveria apontar já para algum lugar? :thinking:

Reply:
@miguelzinh3 tô aguardando o link haha